Update/onik_002.txt

```diff
@@ -657,7 +657,7 @@ void main()
 		   NULL, "\"We're parading around like this, after all.", Line_WaitForInput);
 	PlaySE(3, "s19/03/990300105", 256, 64);
 	OutputLine(NULL, "みんな思うんじゃない？",
-		   NULL, " Don't you all think so, too?", Line_WaitForInput);
+		   NULL, " It'll give everyone a chance to see for themselves.", Line_WaitForInput);
```

Comment:
There seems to be a question mark at the end of the japanese sentence. Can you re-check the meaning?